src/App.jsx

```diff
@@ -1,33 +1,73 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from "react"
+import "./App.css"
+import Inbox from "./components/Inbox"
+import InboxDetails from "./components/InboxDetails"
+import NavBar from "./components/NavBar"
+import Search from "./components/Search"
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [emailData, setEmailData] = useState([])
+
+  useEffect(() => {
+    fetch("https://email-client-api.dev.io-academy.uk/emails")
+      .then((response) => response.json())
+      .then((data) => {
+        setEmailData(data.data)
+      })
+  }, [])
+
+  const [hiddenToggle, setHiddenToggle] = useState("hidden")
```

Comment:
Generally we like to put all of the variable declarations, including state, at the top of the containing function. This is so that we know where to look when adding future code, to see if there is already state for the feature we want to add.

It also makes that state available for every function declared below it in the file - could be important!